apple/RNSVGSvgViewModule.mm

```diff
@@ -90,5 +84,13 @@ - (void)toDataURL:(nonnull NSNumber *)reactTag
   return std::make_shared<facebook::react::NativeSvgViewModuleSpecJSI>(params);
 }
 #endif
+    
+- (dispatch_queue_t)methodQueue {
+    #ifdef RCT_NEW_ARCH_ENABLED
+        return dispatch_get_main_queue();
+    #else
+        return _bridge.uiManager.methodQueue;
+    #endif // RCT_NEW_ARCH_ENABLED
+}
```

Comment:
I checked that and there is some problem when we change `bridgelessEnabled`, I recommend to do that like that:
```suggestion
- (dispatch_queue_t)methodQueue {
    if(self.bridge) {
      return RCTGetUIManagerQueue();
    } else {
      return dispatch_get_main_queue();
    }
}
```